fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -36,9 +36,26 @@ set(FastRTPS_FOUND FALSE)
 find_path(FastRTPS_INCLUDE_DIR
   NAMES fastrtps/)
 
+find_package(fastcdr REQUIRED CONFIG)
+find_package(fastrtps REQUIRED CONFIG)
+
+string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastcdr_DLL_VERSION "${fastcdr_VERSION}")
+string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastrtps_DLL_VERSION "${fastrtps_VERSION}")
```

Comment:
Can you please change the variable name to something without `dll` (since this is not Windows specific code). Maybe `fastcdr_MAJOR_MINOR_VERSION`?